packages/ses/src/module-load.js

```diff
@@ -297,6 +322,63 @@ const memoizedLoadWithErrorAnnotation = async (
   return moduleLoading;
 };
 
+function asyncOverseer() {
+  /** @type {Set<Promise<undefined>>} */
+  const pendingJobs = new Set();
+  /** @type {Array<Error>} */
+  const errors = [];
+
+  /**
+   * Enqueues a job that starts immediately but won't be awaited until drainQueue is called.
+   * 
+   * @template {any[]} T
+   * @param {(...args: T)=>Promise<*>} func
+   * @param {T} args
```

Comment:
@boneskull A few months ago I had trouble specifying what a template extends, but now it seems to work fine with this syntax.